docs/navigate/tools-diagnostics/toc.yml

```diff
@@ -403,7 +403,7 @@ items:
               - name: .NET extensions metrics
                 href: ../../core/diagnostics/built-in-metrics-diagnostics.md
               - name: ASP.NET Core metrics
-                href: ../../core/diagnostics/built-in-metrics-aspnetcore.md
+                href: /aspnet/core/log-mon/metrics/built-in
```

Comment:
Can you make this a contextual TOC link? TOC links aren't supposed to take you away from the current TOC.

Reply:
Thanks @gewarren.  If by "contextual TOC link" you mean linking to a topic in this repository first so that the user then hits a link again in that topic to go to the eventual topic in the other repo, I guess that would either be a link to the Metrics > Overview just above this one which feels a little circular,  or creating a new topic in this repo just to have a link to the actual topic.  

It looks like to get around this problem for the EF metrics, that the EF metrics link was not provided in the TOC and the Metrics > Overview was just relied on where it has a link to the metric topic in the EF repo.  So, I will go that route unless there is something you feel that was better to do.